src/frontend/e2e/setup/global-setup.ts

```diff
@@ -1,14 +1,16 @@
 import { chromium, FullConfig } from "@playwright/test";
 import { getByID, getByTestID } from "../utils/selectors";
-import * as fs from "fs";
+import path from "path";
+import dotenv from 'dotenv';
 
+dotenv.config({path: path.resolve(`.env.${process.env.NODE_ENV}`),});
 
-const username = process.env.USERNAME ?? "";
-const password = process.env.PASSWORD ?? "";
+
+const username: string = process.env.USERNAME as string;
+const password: string = process.env.PASSWORD as string;
```

Comment:
somehow the string casting came back, do you need this to pass linting? If so can you add a todo to address this later or fix the downstream component to accept a different type?
```suggestion
const username: string = process.env.USERNAME;
const password: string = process.env.PASSWORD;
```

Reply:
i'll ignore for now, but let's try not to make type assertions like this in the future

Reply:
Let's remove the assertions before we merge